kapitan/cli.py

```diff
@@ -438,13 +409,15 @@ def secret_reveal(args, ref_controller):
     file_name = args.file
     if file_name is None:
         fatal_error('--file is required with --reveal')
-    if file_name == '-':
-        # TODO deal with RefHashMismatchError or KeyError exceptions
-        out = revealer.reveal_raw_file(None)
-        sys.stdout.write(out)
-    elif file_name:
-        for rev_obj in revealer.reveal_path(file_name):
-            sys.stdout.write(rev_obj.content)
+    try:
+        if file_name == '-':
+            out = revealer.reveal_raw_file(None)
+            sys.stdout.write(out)
+        elif file_name:
+            for rev_obj in revealer.reveal_path(file_name):
+                sys.stdout.write(rev_obj.content)
+    except (RefHashMismatchError, KeyError):
+        logger.exception("Reveal failed for file {name}".format(name=file_name))
```

Comment:
Maybe should not logger.exception here but instead raise a KapitanError with that same message